clientv3/integration/kv_test.go

```diff
@@ -442,7 +442,7 @@ func TestKVGetErrConnClosed(t *testing.T) {
 	go func() {
 		defer close(donec)
 		_, err := cli.Get(context.TODO(), "foo")
-		if err != nil && err != grpc.ErrClientConnClosing {
+		if err != nil && err != context.DeadlineExceeded && err != grpc.ErrClientConnClosing {
 			t.Fatalf("expected %v, got %v", grpc.ErrClientConnClosing, err)
```

Comment:
this needs to be similar to line 795?

Reply:
also this seems wrong. the user does not pass in deadline (context.TODO()). so why deadlineExceeded will be returned? is dialing deadline exceeded in this case or what?